Automattic/newspack-plugin
# feat/campaigns wizard modal

### All Submissions:






### Changes proposed in this Pull Request:



@brianboyer - please have a look. Apart from the UI changes there's some improved messaging, which should help users with dealing with the complexity of this product. 

Before - a popover:

<img width="313" alt="image" src="https://user-images.githubusercontent.com/7383192/126809879-b0dca5bd-c142-4d77-9ace-a68c06b80d60.png">

After - a modal:

<img width="704" alt="image" src="https://user-images.githubusercontent.com/7383192/126809777-b67110c7-1635-4097-b1eb-b0556ed2581a.png">


Closes #926.

### How to test the changes in this Pull Request:

1. Switch to `fix/add-option-setting` branch of `newspack-popups` (https://github.com/Automattic/newspack-popups/pull/588)
1. Visit the Campaigns Wizard, click on the settings icon on a prompt action card
2. Observe a modal being displayed
3. Change the settings, observe the prompt is updated

### Other information:
